Tests/scripts/wait_until_server_ready.py

```diff
@@ -89,7 +99,6 @@ def is_correct_content_installed(ips, content_version, api_key):
                 err_msg += "Server response: {}".format(resp_json)
             print_error(err_msg)
             return False
-    print_color("Content was installed successfully on all of the instances! :)", LOG_COLORS.GREEN)
```

Comment:
keep the print incase of success

Reply:
We already print success for each instance. It is stam here. Plus in the case of release branch it's not even true - it's just misleading.